plugins/boltz/main.go

```diff
@@ -26,7 +27,7 @@ var PluginFlags = []cli.Flag{
 		Name: "boltzurl", Value: DefaultBoltzUrl, Usage: "url of boltz api", Hidden: false,
 	},
 	cli.StringFlag{
-		Name: "boltzdatabase", Value: btcutil.AppDataDir("bolt", false) + "/boltz.db", Usage: "full path to the database file", Hidden: false,
+		Name: "boltzdatabase", Value: btcutil.AppDataDir("bolt", false) + "/boltz.db", Usage: "full writtable path to the database file", Hidden: false,
```

Comment:
"writable" I think, but. I don't like the wording (including mine).
`Path to database file (file will be created if it does not exist yet)` then permission requirement is kind of logical